src/frequenz/sdk/power/_distribution_algorithm.py

```diff
@@ -5,13 +5,20 @@
 
 import logging
 from dataclasses import dataclass
-from math import ceil, floor
+from functools import partial
+from math import isclose
 from typing import Dict, List, NamedTuple, Tuple
 
 from ..microgrid.component import BatteryData, InverterData
 
 _logger = logging.getLogger(__name__)
 
+# A value of 1e-9 is a commonly used absolute tolerance to balance precision
+# and robustness for floating-point numbers comparisons close to zero. Note
+# that this is also the default value for the relative tolerance.
+# For more technical details, see https://peps.python.org/pep-0485/#behavior-near-zero
+_is_close_to_zero = partial(isclose, b=0.0, abs_tol=1e-9)
```

Comment:
Super silly, and I don't think it is important to change now, but I would use the fully qualified name here (for both `math` and `functools`) for 4 reasons:

1. You only use these symbols once, so it is not even more typing
2. It is more clear when you read the code, as they are pretty generic names what you are importing
3. Avoid polluting the global namespace
4. Be more compliant with google style imports

It would be good to have this in mind for future PRs though.

Reply:
Good points, I'll update it now as it is a simple change